independent-projects/arc/runtime/src/main/java/io/quarkus/arc/ArcContainer.java

```diff
@@ -134,6 +134,32 @@ public interface ArcContainer {
      */
     <T> InjectableInstance<T> select(TypeLiteral<T> type, Annotation... qualifiers);
 
+    /**
+     * List all beans matching the required type and qualifiers.
```

Comment:
Maybe document that the list is sorted by bean priority?

Reply:
Well, it has the same semantics as `io.quarkus.arc.All` which is also referenced in the `@see` tag. But ofc I can add it here as well..